### DISCUSSION TITLE
Download Link for Neolink Raspberry Pi 4 64 bit is not working

### DISCUSSION THREAD

**Discussion by melodromo** [thirtythreeforty/neolink#354] [Q&A]
Hi together,

I´m new here and I find the community very great.
I´m trying to use my Reolink Lumus Camera on HA but I need an RTSP bridge, so I want to install the Neolink. I´m struggling with the download of Neolink for Raspeberry Pi 4 64 bit. This link it doesn´t work: https://nightly.link/thirtythreeforty/neolink/workflows/build/master/release-arm64-buster.zip

Am I missing something?

Cheers
melodromo

**Comment by QuantumEntangledAndy** [comment]
As mentioned in another issue. This fork is mostly unmaintained. For updates please see my fork https://github.com/QuantumEntangledAndy/neolink.

For HA though I believe these is an addon here https://github.com/dm82m/hassio-addons/ that you can use. It just uses neolink internally for a bridge.
